lib/token-request.js

```diff
@@ -321,7 +321,7 @@ TokenRequest.prototype._getTokenUsernamePasswordFederated = function(username, p
       return;
     }
 
-    this._performUsernamePasswordForAccessTokenExchange(this._userRealm.federationActiveAuthUrl, username, password, callback);
+    this._performUsernamePasswordForAccessTokenExchange(this._userRealm.federationActiveAuthUrl, constants.WSTrustVersion.UNDEFINED, username, password, callback);
```

Comment:
yep, but my question is, if we don't need to parse the mex, and get only one endpoint back, can we by default assume it is wstrust 13? If not, then I agree with you, we definitely need to know the version here.